skl2onnx/convert.py

```diff
@@ -123,4 +126,4 @@ def convert_sklearn(model, name=None, initial_types=None, doc_string='',
     onnx_model = convert_topology(topology, name, doc_string, target_opset,
                                   options=options)
 
-    return onnx_model
+    return (onnx_model, topology) if intermediate else onnx_model
```

Comment:
can we make this intermediate stuff to a global object which the user can set flag and get the data from it so we don't need alter the API?
BTW, the topology is not a class we want to expose as API, it's better to wrap it as func in a more standard way.

Reply:
I really don't like global objects. It is hard to debug and global variable it usually a bad design. topology is not exposed but azureml.automl is using it. I don't specifically need topology, my goal is to map ONNX nodes scikit-learn operators and the topology has that information. I can extract it and return it instead of the topology but it seems more simple this way.